src/main/java/Circle.java

```diff
@@ -0,0 +1,18 @@
+public class Circle extends Shape {
+
+    int radius;
```

Comment:
Pamiętaj o hermetyzacji  - to pole powinno być private.
https://1024kb.pl/nauka-java/hermetyzacja-java/